doc/source/timeseries.rst

```diff
@@ -883,6 +884,40 @@ under the default business hours (9:00 - 17:00), there is no gap (0 minutes) bet
     # The result is the same as rollworward because BusinessDay never overlap.
     BusinessHour().apply(Timestamp('2014-08-02'))
 
+``BusinessHour`` regards Saturday and Sunday as holidays. To use arbitrary holidays,
+you can use ``CustomBusinessHour`` offset, see :ref:`Custom Business Hour <timeseries.custombusinesshour>`:
+
+.. _timeseries.custombusinesshour:
+
+Custom Business Hour
+~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 0.18.1
+
+The ``CustomBusinessHour`` is a mixture of ``BusinessHour`` and ``CustomBusinessDay`` which
+allows you to specify arbitrary holidays. ``CustomBusinessHour`` works as the same
+as ``BusinessHour`` except that it skips specified custom holidays.
+
+.. ipython:: python
+
+    from pandas.tseries.holiday import USFederalHolidayCalendar
```

Comment:
can do this on another PR, but I think we need to be explict about these mimports (e.g. ``pd.offsets.CustomeBusinessHour``)